plugins/inputs/t128_graphql/README.md

```diff
@@ -36,6 +36,12 @@ The graphql input plugin collects data from a 128T instance via graphQL.
 # [inputs.t128_graphql.extract_tags]
 #   peer-name = "name"
 #   device-interface = "paths/deviceInterface"
+
+## Other tags for filtering data with the desired name as the key (left) and the FULL graphQL 
```

Comment:
These are not supposed to be used for filtering, right? I don't recall how these tags are different from the existing ones. I think it will need to be renamed for clarity, but I can't suggest because I don't fully understand 😄

Reply:
It's unclear at the moment but they are the same except that `other_tags` needs a full path. I talked to Ryan Stizman and he likes `/` at the start to indicate a full path. imo it makes config easier and will help avoid churn on the monitoring-agent plugin too

Reply:
@santi969 now that I see this here, I do think that `other_tags` from a user perspective is a little odd. How about leveraging the `extract_tags` (and heck even `extract_fields`) to do something like this:

```
# [inputs.t128_graphql.extract_tags]
#   peer-name = "name"
#   device-interface = "paths/deviceInterface"
#   router-name = "allRouters/nodes/name"
```

In this case the `router-name` has a "full" path v/s the rest of them have a relative path. We should be able to figure out which is which? Thoughts?

Reply:
Makes sense to me. It'll be a quick fix too